tests/console_test.py

```diff
@@ -0,0 +1,87 @@
+import os
+import sys
+import pytest
+
+import config.main as config
+import tests.mock_tables.dbconnector
+
+from click.testing import CliRunner
+from utilities_common.db import Db
+
+class TestConfigConsoleCommands(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+    
+    def test_console_add_exists(self):
+        runner = CliRunner()
+        db = Db()
+        db.cfgdb.set_entry("CONSOLE_PORT", 1, { "baud_rate" : "9600" })
+
+        # add a console setting which the port has exists
+        result = runner.invoke(config.config.commands["console"].commands["add"], ["1", '--baud', "9600"], obj=db)
+        print(result.exit_code)
+        print(sys.stderr, result.output)
+        assert result.exit_code != 0
+        assert "Trying to add console port setting, which is already exists." in result.output
```

Comment:
Why is this a failure? Do we want to support update configuration of an existing line?

Reply:
Yes, we want to support update configuration for an existing line.